tests/unit/test_env.py

```diff
@@ -42,32 +43,28 @@ def test_env():
                 # value
                 "--env",
                 "SPAM_2=",
-                # "--",
                 tmpdir,
+                "--",
                 "/bin/bash",
                 "-c",
                 # Docker exports all passed env variables, so we can
                 # just look at exported variables.
-                "export; sleep 1",
```

Comment:
Removing this sleep is what verifies that this is (probably) actually fixing the problem.